be/src/exec/hdfs_scanner.cpp

```diff
@@ -432,7 +432,7 @@ StatusOr<bool> HdfsScannerContext::should_skip_by_evaluating_not_existed_slots()
 }
 
 void HdfsScannerContext::append_or_update_partition_column_to_chunk(ChunkPtr* chunk, size_t row_count) {
-    if (partition_columns.size() == 0 || row_count < 0) return;
+    if (partition_columns.size() == 0 || row_count == 0) return;
```

Comment:
don't check this `row_count == 0`.

sometimes even row count is zero, we still have to append or add a partition colunmn to the chunk. for satefy.

Reply:
removed